charts/du-metrics-server/values.yaml

```diff
@@ -1,3 +1,14 @@
+global:
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+  imagePullSecrets:
+    - name: accelleran-secret
+
+  persistentLogLevel: info
```

Comment:
Irrelevant(?) for this PR but there is a lot of unnecessary logging in the du-metrics-server. I think we should either fix that ourselves and create a PR in the GNB repo or make a feature request. I will check with Bart

Reply:
I cleaned it up already. It was part of 3.0.3 I think.